15525 Preserve query properties on tab activation

hschink opened the pull request:
[`* `#15525` Filters and other settings not persisted after activating details pane`](https://www.openproject.org/work_packages/15525)

Commit:
Preserve query properties on tab activation

TeatroIO commented:
I've prepared a stage. <a href='http://fix-15525-filters-and-other-settings-not-persisted-after-activa.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Click to open.</a>

Commit:
Wait for context menu being closed

Commit:
Update context menu version

The new release fixes an issue preventing the context menu from being closed via ESC in Firefox.

Commit:
Fix spec

The spec should actually test closing of the context menu via ESC key. Because
of an error in the angular context menu implementation, the expected behavior
wasn't implemented initially.
